### ISSUE TITLE
How to use ViewBadger?

### ISSUE THREAD

**Issue by mhosman** [NAXAM/bottomnavigationviewex-android-binding#1]
How to add the Badge Lib in order to add badge? I cannot find the solution using the provided NuGet package.

**Comment by tuyen-vuduc** [comment]
Thank you for using our binding library.

I just published the latest nugets for both BottomNavigationViewEx and BadgeView.

Hope you will enjoy using them. Plz give us a star if you like them.

**Comment by mhosman** [comment]
Thanks kanvuduc for your fast answer!! I already give you a star 👍 

I downloaded the updated NuGet package of BottomNavigationViewEx and installed the new BadgeView NuGet Package.

There's any way to write some kind of example into the Readme for BadgeView? For example, how to dynamically set a badge in the Bottom Navigation View when there is a new notification? Example:

![example](https://user-images.githubusercontent.com/5334175/27005616-7578a2b8-4df8-11e7-9963-1c1625715ae6.gif)

I created this example:

```
// Bind view in activity
BottomNavigationViewEx bnve = FindViewById<BottomNavigationViewEx>(Resource.Id.bnve);

// Disable all animations
bnve.enableAnimation(false);
bnve.enableShiftingMode(false);
bnve.enableItemShiftingMode(false);

// Add badge to the last item (fourth - pos 3)
BadgeView bv = new BadgeView(this).SetTextSize(8).Bind(bnve.GetBottomNavigationItemView(3)).SetBadgeCount(2);
```
But i get something like this:
![captura](https://user-images.githubusercontent.com/5334175/27005899-40f28b20-4dfe-11e7-8cec-82b02fc8e29c.PNG)

I don't know why, but the badge is too big by default.

Thank you!!!! Great binding lib!
